### PR TITLE
Sort deprecation clauses alphabetically in :elixir_dispatch

### PR DESCRIPTION
**Pull request by eksperimental**
Hope this helps maintaining the CHANGELOG and Deprecation pages

/cc @whatyouhide

### CODE REVIEW

File: lib/elixir/src/elixir_dispatch.erl [hunk 1]
```diff
@@ -351,64 +351,67 @@ deprecation_message(Warning, Message) ->
...
-  "use Enum.uniq_by/2";
 deprecation('Elixir.Keyword', size, 1) ->
   "use Kernel.length/1";
+deprecation('Elixir.List.Chars', to_char_list, 1) ->
+  "use List.Chars.to_charlist/1";
 deprecation('Elixir.Map', size, 1) ->
   "use Kernel.map_size/1";
 deprecation('Elixir.Stream', uniq, 2) ->
   "use Stream.uniq_by/2";
+%% -- Elixir.String.{l,r}just/2-3
```
Comment (whatyouhide) [inline lib/elixir/src/elixir_dispatch.erl]: I think we can drop this.

File: lib/elixir/src/elixir_dispatch.erl [hunk 2]
```diff
@@ -351,64 +351,67 @@ deprecation_message(Warning, Message) ->
...
+deprecation('Elixir.String', ljust, 2) ->
+  "use String.pad_trailing/2";
+deprecation('Elixir.String', rjust, 2) ->
+  "use String.pad_leading/2";
+deprecation('Elixir.String', ljust, 3) ->
+  "use String.pad_trailing/3 with a binary padding";
+deprecation('Elixir.String', rjust, 3) ->
+  "use String.pad_leading/3 with a binary padding";
+%% -- Elixir.String.*strip/1-2
```
Comment (whatyouhide) [inline lib/elixir/src/elixir_dispatch.erl]: I think we can drop this.

File: lib/elixir/src/elixir_dispatch.erl [hunk 3]
```diff
@@ -351,64 +351,67 @@ deprecation_message(Warning, Message) ->
...
+  "use String.trim_leading/1";
+deprecation('Elixir.String', rstrip, 1) ->
+  "use String.trim_trailing/1";
+deprecation('Elixir.String', strip, 2) ->
+  "use String.trim/2 with a binary second argument";
+deprecation('Elixir.String', lstrip, 2) ->
+  "use String.trim_leading/2 with a binary as second argument";
+deprecation('Elixir.String', rstrip, 2) ->
+  "use String.trim_trailing/2 with a binary as second argument";
+%% --
```
Comment (whatyouhide) [inline lib/elixir/src/elixir_dispatch.erl]: I think we can drop this.

**Comment by eksperimental** [comment]
@whatyouhide comments are there because those lines are grouped, not sorted alphabetically. So I wanted to highlight it somehow.
Should I sort them alphabetically? or simply remove the comments?

**Comment by whatyouhide** [comment]
@eksperimental I would sort them alphabetically.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**